### PR TITLE
Fix for the motech-file-upload directive

### PR DESCRIPTION
**Pull request by jredlarski**
Fixed wrong url in directive.
Fixed wrong icon in html file.

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by jredlarski** [comment]
@shanmukhm Thanks for reporting these issues. Could you please check if it is ok now?

**Comment by sebbrudzinski** [comment]
test this please

File: src/common/motech-file-upload/motech-file-upload.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="form-control input-auto input-large fileinput-exists" data-trigger="fileinput">
-    <span class="glyphicon glyphicon-file"></span>&nbsp;
+    <span class="icon-file"></span>&nbsp;
```
Comment (shanmukhm) [inline src/common/motech-file-upload/motech-file-upload.html]: Everything is alright. Just add this in 'base.icons.scss'. (It's working in my repo as I already have this property defined)
```
.icon-file{
    @include icon(file);
 }
```

**Pull request merged**